_includes/md/mdhost.php

```diff
@@ -0,0 +1,67 @@
+<?php
+  require_once('includes/template.php');
+  require_once(__DIR__ . '/../../vendor/erusev/parsedown/Parsedown.php');
+  require_once(__DIR__ . '/../../vendor/erusev/parsedown-extra/ParsedownExtra.php');
+  use Keyman\Site\Common\KeymanHosts;
+
+  $pagetitle = 'TODO';
+
+  if(!isset($_REQUEST['file'])) {
+    die('Require file parameter');
+  }
+
+  $file = realpath(__DIR__ . '/../../') . DIRECTORY_SEPARATOR . $_REQUEST['file'];
+  $contents = trim(file_get_contents($file));
+  $contents = str_replace("\r\n", "\n", $contents);
+
+  // Regex Keyman hosts
+  $contents = str_replace("\$api.keyman.com", KeymanHosts::Instance()->api_keyman_com, $contents);
+  $contents = str_replace("\$donate.keyman.com", KeymanHosts::Instance()->donate_keyman_com, $contents);
+  $contents = str_replace("\$help.keyman.com", KeymanHosts::Instance()->help_keyman_com, $contents);
+  $contents = str_replace("\$keyman.com", KeymanHosts::Instance()->keyman_com, $contents);
```

Comment:
There are a bunch of domains missing here (s.keyman.com, downloads.keyman.com, keymanweb.com, r.keymanweb.com, blog.keyman.com, etc...)

I think we should define this as an instance function in the `KeymanHosts` class, e.g. `fixupHostReferences`. Then in the .md we always use the *live* site references (e.g. `https://api.keyman.com/`) and have:

```suggestion
  KeymanHosts::Instance()->fixupHostReferences($contents);
```

This pattern can then also be used on help.keyman.com which already has Markdown.

My reasoning for this is that it helps the editor of the file if they have working links in their preview (of course, if the link is to a page only on staging, it won't work, but that's going to be the exception rather than the rule).